---
layout: vanilla
permalink: /data/
description: "Downloads for Mouse Atlas."
modified: 2018-01-13
tags: [worm, manual, vignette, atlas]
---
<!DOCTYPE html>
<html lang= "{{ page.lang | default: site.lang | default: "en" }}">

  {% include head.html %}

  <!-- See here for how to use the TOC control:
  https://afeld.github.io/bootstrap-toc/ -->
  <body  data-spy= "scroll" data-target= "#toc">
    {% include header.html %}

    <div class= "container">
      <div class= "row">
        <!-- sidebar, which will move to the top on a small screen -->
        <div class= "col-sm-4 sidenav">
          <nav id= "toc" data-spy= "affix" data-toggle= "toc"></nav>
        </div>
        <!-- main content area -->
        <div class= "col-sm-8">

<!--           <h2>Bulk Download</h2>
          <p>Files may be downloaded individually below or in a single compressed archive. Documentation for all files is provided in the sections below.</p>

          <a class="btn btn-primary" href="http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/mouse_atac_data_release.tar.gz" role="button"><b>Download All (mouse_atac_data_release.tar.gz; 37GB)</b></a>

<div class="bigspacer"></div>

          <p>The bulk download can be unpacked with OSX or windows utilities or on command line via:</p>

          <pre><code class="language-bash" data-lang="bash">tar -zxvf mouse_atac_data_release.tar.gz</code></pre>

<div class="hugespacer"></div>
 -->
   <h2>Release Updates</h2>
  <ul>
    <li><code>08/14/2018</code> adds <code>UCSC Trackhub and BigWigs</code> section which provides documentation on new bigWig files and a UCSC Trackhub for visualization.</li>
    <li><code>08/10/2018</code> adds another metadata file option, <code>cell_metadata.tissue_freq_filtered.txt</code>, which removes cells in each tissue belonging to a <code>cell_label</code> that accounts for less than 0.5% of the cells in that tissue. These very low frequency labels are often not cell types expected their respective tissues and could be due to slight imperfections in clustering, for example.</li>
    <li><code>08/06/2018</code>:
      <ul>
        <li>Switches any references to Colon to LargeIntestine and Intestine to SmallIntestine. This impacts any file that lists a tissue name, but all files were previously consistent with one another and usable.</li>
        <li>Updates specificity score section including added tutorial input files and minor updates to existing gene-level specificity scores.</li>
        <li>Adds a <code>BAM Files</code> for BAM downloads.</li>
      </ul>
    <li><code>08/02/2018</code>: Initial release.</li>
  </ul>
  <div class="hugespacer"></div>


          <h2>ATAC Matrices</h2>

          <p>Similar to sc-RNA-seq, sci-ATAC-seq data is typically analyzed in sparse <code>peak</code> (row) by <code>cell (column)</code> matrices. The first set we provide are binarized counts. The second set has rare peaks filtered out and is then normalized with TFIDF to allow for input to  PCA/TSNE, for example. Note that only cells in our final QC filtered set are included.</p>

          <p>See <a href="{{ site.baseurl }}/docs/">tutorials</a> for examples of how to read these formats into <code>R</code> or <code>python</code> along with documentation on lots of other downstream analysis.</p>

          <div class= "panel panel-info">
            <div class= "panel-heading">
              <h3 class= "panel-title" data-toc-skip>Matrix Formats</h3>
            </div>
            <div class= "panel-body">
            <p>In general, we provide two formats for all matrices:</p>
            <table class = "table">
              <thead>
                <tr>
                  <th>File Type</th>
                  <th>Description</th>
                </tr>
              </thead>

              <tbody>
                <tr>
                  <th><code>.mtx.gz</code></th>
                  <th><p>Same as format generated by 10X Genomics <code>cellranger</code> pipeline (matrix market format). <code>.mtx</code> files are provided with <code>.txt</code> files containing peak and cell IDs that correspond to the rows and columns of the matrix, respectively.</p></th>
                </tr>

                <tr>
                  <th><code>.rds</code></th>
                  <th><p>RDS format that can be read into R directly with <code>readRDS</code> function.</p></th>
                </tr>
              </tbody>
            </table>
            </div>
          </div>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filte
red.mtx.gz" download>atac_matrix.binary.qc_filtered.mtx.gz</a></th>
  <th>08/02/18</th>
  <th>1.1GB</th>
  <th>Binarized peak by cell matrix in matrix market format.</th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filtered.cells.txt" download>atac_matrix.binary.qc_filtered.cells.txt</a></th>
  <th>08/02/18</th>
  <th>2.9MB</th>
  <th>Cell IDs (columns) of binarized matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filtered.peaks.txt" download>atac_matrix.binary.qc_filtered.peaks.txt</a></th>
  <th>08/02/18</th>
  <th>10.0MB</th>
  <th>Peak IDs (rows) of binarized matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filtered.rds" download>atac_matrix.binary.qc_filtered.rds</a></th>
  <th>08/02/18</th>
  <th>1.0GB</th>
  <th>Binarized peak by cell matrix in RDS format.</th>
</tr>

<tr>
  <th></th>
  <th></th>
  <th></th>
  <th></th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filter
ed.mtx.gz" download>atac_matrix.tfidf.qc_filtered.mtx.gz</a></th>
  <th>08/02/18</th>
  <th>3.4GB</th>
  <th>TFIDF normalized peak by cell matrix in matrix market format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filtered.cells.txt" download>atac_matrix.tfidf.qc_filtered.cells.txt</a></th>
  <th>08/02/18</th>
  <th>2.9MB</th>
  <th>Cell IDs (columns) of TFIDF matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filtered.peaks.txt" download>atac_matrix.tfidf.qc_filtered.peaks.txt</a></th>
  <th>08/02/18</th>
  <th>3.8MB</th>
  <th>Peak IDs (rows) of TFIDF matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filtered.rds" download>atac_matrix.tfidf.qc_filtered.rds</a></th>
  <th>08/02/18</th>
  <th>5.4GB</th>
  <th>TFIDF normalized peak by cell matrix in RDS format.</th>
</tr>

                </tbody>
              </table>




<div class="hugespacer"></div>

          <h2>Activity Score Matrices</h2>
          <p>We also report "gene activity scores", where a single number is calculated based on a weighted combination of proximal and distal sites for each gene (see manuscript for details; both quantitative and binarized calculations provided below). Unlike the ATAC matrices above, these are in <code>gene (row)</code> by <code>cell (column)</code> format.</p>

         <div class= "panel panel-warning">
            <div class= "panel-heading">
            <h3 class= "panel-title" data-toc-skip>Important: Size Factor Normalization</h3>
            </div>
            <div class= "panel-body">
<p>Quantitative scores provided here are not normalized by size factors, so you may apply size factor normalization to these values if needed.</p>
            </div>
          </div>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.quantitative.mtx.gz" download>activity_scores.quantitative.mtx.gz</a></th>
  <th>08/02/18</th>
  <th>754.1MB</th>
  <th>Quantitative gene activity score matrix in matrix market format.</th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.quantitative.cells.txt" download>activity_scores.quantitative.cells.txt</a></th>
  <th>08/02/18</th>
  <th>2.9MB</th>
  <th>Cell IDs (columns) of quantitative gene activity score matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.quantitative.genes.txt" download>activity_scores.quantitative.genes.txt</a></th>
  <th>08/02/18</th>
  <th>150.4KB</th>
  <th>Gene names (common names; columns) of quantitative gene activity score matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.quantitative.rds" download>activity_scores.quantitative.rds</a></th>
  <th>08/02/18</th>
  <th>664.0MB</th>
  <th>Quantitative gene activity score matrix in RDS format.</th>
</tr>

<tr>
  <th></th>
  <th></th>
  <th></th>
  <th></th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.binarized.mtx.gz" download>activity_scores.binarized.mtx.gz</a></th>
  <th>08/02/18</th>
  <th>218.1MB</th>
  <th>Binarized gene activity score matrix in matrix market format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.binarized.cells.txt" download>activity_scores.binarized.cells.txt</a></th>
  <th>08/02/18</th>
  <th>2.9MB</th>
  <th>Cell IDs (columns) of binarized gene activity score matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.binarized.genes.txt" download>activity_scores.binarized.genes.txt</a></th>
  <th>08/02/18</th>
  <th>150.4KB</th>
  <th>Gene names (common names; columns) of binarized gene activity score matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/activity_score_matrices/activity_scores.binarized.rds" download>activity_scores.binarized.rds</a></th>
  <th>08/02/18</th>
  <th>151.9MB</th>
  <th>Binarized gene activity score matrix in RDS format.</th>
</tr>

                </tbody>
              </table>



<div class="hugespacer"></div>

          <h2>Metadata</h2>
          <p>For all cells and peaks used in our QC filtered set, we report tables of metadata including information about tissue source, cell type assignment, TSNE coordinates, cluster assignments, etc. for cells, and intersections with genes (TSS only) for peaks.</p>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/metadata/cell_metadata.txt" download
                >cell_metadata.txt</a></th>
                  <th>08/06/18</th>
                  <th>13.3MB</th>
                  <th>Metadata for cells in TSV format, including several features such as TSNE coordinates, cluster assignments, and cell type assignments.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#cell-metadata-format" role="button" aria-expanded="false" aria-controls="cell-metadata-format">format details</a>
                    <div class="collapse" id="cell-metadata-format">
                      <ul>
                        <li><code>cell</code>: cell barcode (combined and corrected)</li>
                        <li><code>tissue</code>: the tissue that this cell originated from</li>
                        <li><code>tissue_replicate</code>: same as tissue, but each replicate has a unique id</li>
                        <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                        <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                        <li><code>tsne_1</code>: t-SNE1 coordinate in initial t-SNE</li>
                        <li><code>tsne_2</code>: t-SNE2 coordinate in initial t-SNE</li>
                        <li><code>subset_tsne1</code>: t-SNE1 coordinate in iterative t-SNE</li>
                        <li><code>subset_tsne2</code>: t-SNE2 coordinate in iterative t-SNE</li>
                        <li><code>id</code>: combined ID for major + iterative cluster assignment</li>
                        <li><code>cell_label</code>: assigned cell type</li>
                      </ul>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/metadata/cell_metadata.tissue_freq_filtered.txt" download
                >cell_metadata.tissue_freq_filtered.txt</a></th>
                  <th>08/10/18</th>
                  <th>13.0MB</th>
                  <th>Same as <code>cell_metadata.txt</code>, but removes cells in each tissue belonging to a <code>cell_label</code> that accounts for less than 0.5% of the cells in that tissue. These very low frequency labels are often not cell types expected their respective tissues and could be due to slight imperfections in clustering, for example. Provided matrices would need to be subsetted to match this set of cells if using this metadata.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/metadata/peak_promoter_intersections.txt" download>peak_promoter_intersections.txt</a></th>
                  <th>08/02/18</th>
                  <th>6.9MB</th>
                  <th>Metadata for peaks-intersected TSS pairs in TSV format
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#peak-metadata-format" role="button" aria-expanded="false" aria-controls="peak-metadata-format">format details</a>
                    <div class="collapse" id="peak-metadata-format">
                      <ul>
                        <li><code>peak_id</code>: ID of peak (chr_start_end)</li>
                        <li><code>peak_chr</code>: chromosome for peak location</li>
                        <li><code>peak_start</code>: start coordinate for peak location</li>
                        <li><code>peak_end</code>: end coordinate for peak location</li>
                        <li><code>ensembl_id</code>: ensembl ID for intersected gene TSS</li>
                        <li><code>gene_short_name</code>: common name for intersected gene TSS</li>
                        <li><code>ensembl_transcript_id</code>: ensembl ID for intersected transcript TSS</li>
                        <li><code>biotype</code>: biotype for intersected gene TSS</li>
                        <li><code>strand</code>: strand for intersected gene TSS</li>
                      </ul>
                    </div>
                  </th>
                </tr>


                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/metadata/cell_type_assignments.xlsx" download>cell_type_assignments.xlsx</a></th>
                  <th>08/06/18</th>
                  <th>134.1KB</th>
                  <th>Excel document with three tabs <code>expected cell types</code>, <code>cell type markers</code>, and <code>Cell type assignments</code> that contain a pairs of tissues and expected cell types, a list of positive markers for each cell type, and the table of cell type assignments with extra details about assignment criteria when applicable, respectively. This is meant to document justification for cell type assignments provided in <code>cell_metadata.txt</code> above.
                      <a class="btn btn-info btn-xs" data-toggle="collapse" href="#marker-sheet-format" role="button" aria-expanded="false" aria-controls="marker-sheet-format">format details</a>
                      <div class="collapse" id="marker-sheet-format">
                      <code>expected cell types</code>
                        <ul>
                          <li><code>tissue</code>: the tissue in the cell type tissue pair</li>
                          <li><code>expected_cell_type</code>: the cell type in the tissue cell type pair</li>
                        </ul>

                      <code>cell type markers</code>
                        <ul>
                          <li><code>Cell type</code>: the cell type from above table</li>
                          <li><code>subset_cluster</code>: a list of positive markers for that cell type</li>
                        </ul>

                      <code>Cell type assignments</code>
                        <ul>
                          <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                          <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                          <li><code>pipeline_cell_type</code>: broad automatic assignment made by classifier (see manuscript).</li>
                          <li><code>manual_cell_type</code>: a broad assigned cell type where applicable</li>
                          <li><code>cell_label</code>: assigned cell type (same as in cell metadata file)</li>
                          <li><code>notes</code>: extra notes about assignment criteria where applicable</li>
                        </ul>

                      </div>
                  </th>
                </tr>
                </tbody>
              </table>




<div class="hugespacer"></div>

          <h2>Differential Accessibility</h2>
          <p>We report results from differential accessibility (DA) tests performed between each cluster of cells (final iterative clusters) and a set of 2K sampled cells. See manuscript for details. These are reported using both the binarized ATAC matrix (contains all peaks) and the binarized gene activity score matrix (contains a single entry per gene).</p>


          <div class= "panel panel-info">
            <div class= "panel-heading">
            <h3 class= "panel-title" data-toc-skip>DA Test Format</h3>
            </div>
            <div class= "panel-body">
            <p>The following columns are provided in each DA test file. Note that files contain one entry per combination of <code>cluster</code>, <code>subset_cluster</code>, and <code>peak/gene_short_name</code>.</p>
            <table class = "table">
              <thead>
                <tr>
                  <th>Column</th>
                  <th>Description</th>
                </tr>
              </thead>

              <tbody>
                <tr>
                  <th><code>status</code></th>
                  <th>status of successful test completion reported by monocle (OK for all in this case)</th>
                </tr>

                <tr>
                  <th><code>family</code></th>
                  <th>the distribution family used by monocle (binomialff in this case)</th>
                </tr>

                <tr>
                  <th><code>pval</code></th>
                  <th>Uncorrected P-value returned by monocle.</th>
                </tr>

                <tr>
                  <th><code>beta</code></th>
                  <th>Beta derived from the model returned by monocle. The coefficient of the term noting cluster membership. Negative values indicate less accessibility within the specified cluster than the 2K sampled set.</th>
                </tr>

                <tr>
                  <th><code>qval</code></th>
                  <th>Q-value returned by monocle corrected for all tests performed for the specified cluster.</th>
                </tr>

                <tr>
                  <th><code>peak/gene_short_name</code></th>
                  <th>For ATAC matrix this is the peak ID of the peak being tested (chr_start_stop; header <code>peak</code>). For activity score matrix, this is the common gene name of the gene being tested (header <code>gene</code>).</th>
                </tr>

                <tr>
                  <th><code>cluster</code></th>
                  <th>Cluster assignment in initial t-SNE.</th>
                </tr>

                <tr>
                  <th><code>subset_cluster</code></th>
                  <th>Cluster assignment in iterative t-SNE space.</th>
                </tr>

              </tbody>
            </table>
            </div>
          </div>


            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/da_tests/atac_matrix.binary.da_results.txt" download>atac_matrix.binary.da_results.txt</a></th>
  <th>08/02/18</th>
  <th>2.4GB</th>
  <th>DA test results from ATAC matrix in TSV format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/da_tests/atac_matrix.binary.da_results.sig_open.txt" download>atac_matrix.binary.da_results.sig_open.txt</a></th>
  <th>08/02/18</th>
  <th>118.0MB</th>
  <th>DA test results from ATAC matrix for peaks with significant positive betas in TSV format.</th>
</tr>

<tr>
  <th></th>
  <th></th>
  <th></th>
  <th></th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/da_tests/activity_scores.binarized.da_results.txt" download>activity_scores.binarized.da_results.txt</a></th>
  <th>08/02/18</th>
  <th>83.5MB</th>
  <th>DA test results from binarized gene activity score matrix in TSV format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/da_tests/activity_scores.binarized.da_results.sig_open.txt" download>activity_scores.binarized.da_results.sig_open.txt</a></th>
  <th>08/02/18</th>
  <th>14.2MB</th>
  <th>DA test results from binarized gene activity score matrix for genes with significant positive betas in TSV format.</th>
</tr>
                </tbody>
              </table>


<div class="hugespacer"></div>

          <h2>Specificity Scores</h2>
          <p>We report specificity scores to rank elements by their restricted accessiblity within each of our clusters (see manuscript for details). Only sites that had significant specificity scores at our empirically determined false discovery rate threshold are reported.</p>

          <p>These are provided in both Excel format and text format to allow browsing of results.</p>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
<!-- <tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/specificity_scores.by_cluster.xlsx" download>specificity_scores.by_cluster.xlsx</a></th>
  <th>02/05/2018</th>
  <th>15.0MB</th>
  <th>Excel formatted specificity scores. There is one tab in the document for each cluster, each of which contains a table with specificity scores. Only sites deemed to have signficiant specificity scores are reported in this particular file (see methods). <a class="btn btn-info btn-xs" data-toggle="collapse" href="#specificity-excel-format" role="button" aria-expanded="false" aria-controls="specificity-excel-format">format details</a>
                      <div class="collapse" id="specificity-excel-format">
                        <ul>
                          <li><code>site</code>: ID of peak (chr:start-end)</li>
                          <li><code>beta</code>: Beta derived from the model returned by monocle. The coefficient of the term noting cluster membership. Negative values indicate less accessibility within the specified cluster than the 2K sampled set.</li>
                          <li><code>pval</code>: P-value returned by monocle corrected for all tests performed for the specified cluster.</li>
                          <li><code>qval</code>: P-value corrected for multiple hypothesis testing.</li>
                          <li><code>specificity_score</code>: specificity of accessibility for this peak/cluster combination (see methods)</li>
                        </ul>
                      </div>
  </th>
</tr> -->

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/peak_specificity_scores.long_format.txt.gz" download>peak_specificity_scores.long_format.txt.gz</a></th>
  <th>08/02/2018</th>
  <th>568.7MB</th>
  <th>Long format file with specificity scores for all sites across all clusters. <a class="btn btn-info btn-xs" data-toggle="collapse" href="#specificity-format" role="button" aria-expanded="false" aria-controls="specificity-format">format details</a>
                      <div class="collapse" id="specificity-format">
                        <ul>
                          <li><code>specificity_score</code>: specificity of accessibility for this peak/cluster combination (see methods)</li>
                          <li><code>site</code>: ID of peak (chr_start_end)</li>
                          <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                          <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                          <li><code>cluster_name</code>: an ID combining cluster and subset_cluster into one string.</li>
                        </ul>
                      </div>
  </th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/peak_specificity_scores.long_format.rds" download>peak_specificity_scores.long_format.rds</a></th>
  <th>08/02/2018</th>
  <th>496.2MB</th>
  <th>Same as above in RDS format.</th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/gene_specificity_scores.long_format.txt.gz" download>gene_specificity_scores.long_format.txt.gz</a></th>
  <th>08/06/2018</th>
  <th>17.9MB</th>
  <th>Same as <code>peak_specificity_scores.long_format.txt.gz</code>, but gene-level specificity calculated from activity scores. <code>site</code> column replaced with <code>gene_short_name</code>.</th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/gene_specificity_scores.long_format.rds" download>gene_specificity_scores.long_format.rds</a></th>
  <th>08/06/2018</th>
  <th>12.6MB</th>
  <th>Same as above in RDS format.</th>
</tr>


<!-- And the significant sites -->
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/peak_specificity_scores_sigsites.long_format.txt.gz" download>peak_specificity_scores_sigsites.long_format.txt.gz</a></th>
  <th>08/02/2018</th>
  <th>3.8MB</th>
  <th>Same as <code>peak_specificity_scores.long_format.txt.gz</code>, but filtered to significantly specific sites (see methods).</th>
</tr>

<tr>
    <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/peak_specificity_scores_sigsites.long_format.rds" download>peak_specificity_scores_sigsites.long_format.rds</a></th>
  <th>08/02/2018</th>
  <th>4.5MB</th>
  <th>Same as above in RDS format.</th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/gene_specificity_scores_sigsites.long_format.txt.gz" download>gene_specificity_scores_sigsites.long_format.txt.gz</a></th>
  <th>08/06/2018</th>
  <th>626.1KB</th>
  <th>Same as <code>gene_specificity_scores.long_format.txt.gz</code>, but filtered to significantly specific genes (see methods). </th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/gene_specificity_scores_sigsites.long_format.rds" download>gene_specificity_scores_sigsites.long_format.rds</a></th>
  <th>08/06/2018</th>
  <th>891.0KB</th>
  <th>Same as above in RDS format.</th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Specificity_Scores/specificity_data.tar.gz" download>specificity_data.tar.gz</a></th>
  <th>08/06/2018</th>
  <th>166.4MB</th>
  <th>Data files necessary to complete the tutorial on specificity scoring.</th>
</tr>

                </tbody>
              </table>


<div class="hugespacer"></div>

          <h2>Comparisons to scRNA-seq Datasets</h2>

          <p>In our manuscript we also examine similarity between our sci-ATAC-seq dataset and several sc-RNA-seq datasets. We do this using a cluster-level correlation-based approach and a cell-by-cell KNN-based approach. Both used activity scores as calculated by Cicero as input (see above). Here we provide the cluster-level correlations for each dataset/tissue and the cell-by-cell KNN results for each dataset we have compared to.</p>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/scrna_seq_comparisons/knn_results.txt" download>knn_results.txt</a></th>
  <th>08/06/2018</th>
  <th>8.0MB</th>
  <th>KNN results listing assignments of each cell in our study in TSV format. Some cells will be missing and designated as "NA" due to the filtering of very low frequency cell type assignments within each tissue and thresholds on the number of non-zero peaks per cell as described in methods.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#knn-format" role="button" aria-expanded="false" aria-controls="knn-format">format details</a>
                      <div class="collapse" id="knn-format">
                        <ul>
                          <li><code>cell</code>: cell barcode (combined and corrected)</li>
                          <li><code>tissue</code>: the tissue in the cell type tissue pair</li>
                          <li><code>cell_label</code>: assigned cell type in our study</li>
                          <li><code>assigned_cell_label_microwell</code>: Label assigned from MCA dataset for applicable cells (Han et al.). See columns below for explanation of "NA" values.</li>
                          <li><code>assigned_cell_label_tm</code>: Label assigned from Tabula Muris Consortium dataset for applicable cells. See columns below for explanation of "NA" values.</li>
                          <li><code>label_status_microwell</code>: In cases where labels assigned from microwell dataset are "NA" this column provides a reason why. Entries with non-NA labels assigned with have "assigned" in this column. NA valuesin in the <code>assigned_cell_label_microwell</code> column can be NA due to tissues not overlapping between the two datasets (not_compared), a cell not being included in a comparison due it not meeting our thresholds for number of non-zero sites (filtered_by_depth), or KNN neighbors not providing a clear majority label (low_support).</li>
                          <li><code>label_status_tm</code>: same as label_status_microwell but referring to the assigned Tabula Muris label.</li>
                        </ul>
                      </div>
    </th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/scrna_seq_comparisons/correlation_results.txt" download>correlation_results.txt</a></th>
  <th>08/06/2018</th>
  <th>201.9KB</th>
  <th>Cluster-level Spearman correlations as a metric of similarity between each of our chromatin profiles and cell type assignments made in each of the same sc-RNA-seq studies in TSV format.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#correlation-scrnaseq-format" role="button" aria-expanded="false" aria-controls="correlation-scrnaseq-format">format details</a>
                      <div class="collapse" id="correlation-scrnaseq-format">
                        <ul>
                          <li><code>id</code>: combined ID for major + iterative cluster assignment</li>
                          <li><code>tissue</code>: the tissue in the cell type tissue pair</li>
                          <li><code>cell_label</code>: assigned cell type in our study for the specified cluster</li>
                          <li><code>scrna_seq_dataset</code>: The sc-RNA-seq study used to compute the correlation. One of "tabula_muris" or "microwell".</li>
                          <li><code>scrna_seq_label</code>: Label assigned from sc-RNA-seq dataset.</li>
                          <li><code>correlation</code>: Spearman correlation between the activity scores and expression values for genes with variable gene expression as described in manuscript.</li>
                        </ul>
                      </div>
    </th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/scrna_seq_comparisons/microwell_truncated_labels.txt" download>microwell_truncated_labels.txt</a></th>
  <th>08/02/2018</th>
  <th>6.6KB</th>
  <th>As described in our manuscript, to facilitate comparisons to the MCA dataset, we made minor modifications to the labels provided in this study (see methods for details). This file provides original labels used in the MCA paper and then the set of labels that we used in TSV format.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#mca-mapping-format" role="button" aria-expanded="false" aria-controls="mca-mapping-format">format details</a>
                      <div class="collapse" id="mca-mapping-format">
                        <ul>
                          <li><code>mca_label</code>: the original label for this cell provided in the MCA data release</li>
                          <li><code>mca_label.modified</code>: the modified label that we used for comparisons in the manuscript</li>
                        </ul>
                      </div>
    </th>
</tr>


                </tbody>
              </table>








<div class="hugespacer"></div>

          <h2>Cicero Maps</h2>

          <p>We have also run Cicero (<a href="https://cole-trapnell-lab.github.io/cicero-release/">Pliner <i>et al.</i></a>), which connects regulatory elements to their target genes using coaccessibility as a measure of connectedness, as measured by sci-ATAC-seq. We have generated Cicero maps for each cluster in the dataset. Maps and peak sets are combined into single files with columns to indicate the cluster and subset_cluster entries correspond to.</p>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Cicero_Maps/master_cicero_conns.txt" download>master_cicero_conns.txt</a></th>
  <th>01/11/2018</th>
  <th>3.1GB</th>
  <th>Cicero was run on the peak sets provided below (accessible in at least 1% of cells for a given cluster). This is the resulting map provided in TSV format.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#cicero-map-format" role="button" aria-expanded="false" aria-controls="cicero-map-format">format details</a>
                      <div class="collapse" id="cicero-map-format">
                        <ul>
                          <li><code>Peak1</code>: Peak ID (see below) for first peak in connection.</li>
                          <li><code>Peak2</code>: Peak ID (see below) for second peak in connection.</li>
                          <li><code>coaccess</code>: coaccessibility score for connection. Higher values indicate stronger signal. See <a href="https://www.biorxiv.org/content/early/2017/07/20/166066">Pliner <i>et al.</i></a> for details.</li>
                          <li><code>peak1.isproximal</code>: Peak1 is proximal if it has overlap with 5kb upstream and 1kb downstream of any TSS (set to "Yes" if proximal).</li>
                          <li><code>peak1.tss.gene_name</code>: Common gene name of the proximal TSS overlapping Peak1</li>
                          <li><code>peak1.tss.gene_id</code>: Ensemble gene id of the proximal TSS overlapping Peak1</li>
                          <li><code>peak2.isproximal</code>: same as for peak1</li>
                          <li><code>peak2.tss.gene_name</code>: same as for peak1</li>
                          <li><code>peak2.tss.gene_id</code>: same as for peak1</li>
                          <li><code>conn_type</code>: one of proximal_proximal, distal_proximal, distal_
distal to describe the combination of proximal and distal status of Peak1 and Peak2.</li>
                          <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                          <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                          <li><code>subcluster</code>: combined cluster ID (cluster + subset cluster)</li>
                        </ul>
                      </div>
    </th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Cicero_Maps/master_cicero_conns.rds" download>master_cicero_conns.rds</a></th>
  <th>01/11/2018</th>
  <th>286.3MB</th>
  <th>Same as above in RDS format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Cicero_Maps/open_sites_0.01.txt" download>open_sites_0.01.txt</a></th>
  <th>01/12/2018</th>
  <th>455.3MB</th>
  <th>Peaks that served as input to Cicero in TSV format (accessible in >=1% of cells in a given cluster).
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#cicero-peak-format" role="button" aria-expanded="false" aria-controls="cicero-peak-format">format details</a>
                      <div class="collapse" id="cicero-peak-format">
                        <ul>
                          <li><code>site</code>: ID of peak (chr_start_end)</li>
                          <li><code>num_cells_expressed</code>: number of cells in cluster with at least one read overlapping this site</li>
                          <li><code>fraction_cells_expressed</code>: fraction of cells in cluster with at least one read overlapping this site</li>
                          <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                          <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                        </ul>
                      </div>
    </th>

</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Cicero_Maps/open_sites_0.01.rds" download>open_sites_0.01.rds</a></th>
  <th>01/11/2018</th>
  <th>91.5MB</th>
  <th>Same as above in RDS format.</th>
</tr>
                </tbody>
              </table>


<div class="hugespacer"></div>

          <h2>Basset Results</h2>

          We have also trained convolutional neural network (CNN) models with Basset (<a href="https://github.com/davek44/Basset">Github</a>; <a href="http://genome.cshlp.org/content/early/2016/05/03/gr.200535.115.abstract">Kelley <i>et al.</i></a>) to find motifs that distinguish our clusters from one another. Here we provide results relevant to interpretation of these motifs as well as the actual models generated by Basset so they may be used in downstream analyses.

          To interpret the "filters" in the first layer of the CNN model, we utilize common tools for interpreting PWMs such as TomTom and MEME from the <a href="http://meme-suite.org/">MEME suite</a> in conjunction with the <a href="hocomoco11.autosome.ru">Hocomoco PWM database</a>.

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/training_params.txt" download
>training_params.txt</a></th>
  <th>01/11/2018</th>
  <th>286.0B</th>
  <th>Parameters used to train the CNN using Basset. See Basset documentation.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/trained_model.th" download>trained_model.th</a></th>
  <th>01/11/2018</th>
  <th>88.0MB</th>
  <th>CNN model trained using Basset. See Basset documentation.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/train_test_data.h5" download>train_test_data.h5</a></th>
  <th>01/11/2018</th>
  <th>4.3GB</th>
  <th>Data used to train and test the CNN in the <code>hdf5</code> format. See Bassett documentation.</th>
</tr>

<tr>
  <th></th>
  <th></th>
  <th></th>
  <th></th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/filter_influences.txt" download>filter_influences.txt</a></th>
  <th>01/11/2018</th>
  <th>1.2MB</th>
  <th>Filter influences as calculated using the CNN model in our manuscript in TSV format.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#influences-format" role="button" aria-expanded="false" aria-controls="influences-format">format details</a>
                      <div class="collapse" id="influences-format">
                        <ul>
                          <li><code>filter</code>: name of filter PWM</li>
                          <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                          <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                          <li><code>infl</code>: Influence score. See manuscript.</li>
                          <li><code>filter_id</code>: Numeric ID for the filter</li>
                        </ul>
                      </div>
    </th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/filter_sd_mean.txt" download>filter_sd_mean.txt</a></th>
  <th>01/11/2018</th>
  <th>16.5KB</th>
  <th>Contains summary statistics for filter activity over the test data.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#filter-stats-format" role="button" aria-expanded="false" aria-controls="filter-stats-format">format details</a>
                      <div class="collapse" id="filter-stats-format">
                        <ul>
                          <li><code>filter</code>: name of filter PWM</li>
                          <li><code>ic</code>: information content of the filter over all clusters</li>
                          <li><code>mean</code>: mean influence of the filter over all clusters</li>
                          <li><code>sd</code>: standard deviation for influence of the filter over all clusters</li>
                        </ul>
                      </div>
    </th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/filters_meme.txt" download>filters_meme.txt</a></th>
  <th>01/11/2018</th>
  <th>343.2KB</th>
  <th>PWMs for all the first layer filters in the trained CNN (used for interpretation of filters). See MEME documentation.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/tomtom_hits.txt" download>tomtom_hits.txt</a></th>
  <th>01/11/2018</th>
  <th>247.8KB</th>
  <th>Filters matched to known motifs using TomTom with value < 0.1.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#tomtom-format" role="button" aria-expanded="false" aria-controls="tomtom-format">format details</a>
                      <div class="collapse" id="tomtom-format">
                        <ul>
                          <li><code>#Query ID</code>: name of filter PWM</li>
                          <li><code>Target ID</code>: motif name in Hocomoco database</li>
                          <li><code>Optimal offset</code>: See TomTom documentation</li>
                          <li><code>Overlap</code>: See TomTom documentation</li>
                          <li><code>Query consensus sequence</code>: See TomTom documentation</li>
                          <li><code>Target consensus sequence</code>: See TomTom documentation</li>
                          <li><code>Orientation</code>: See TomTom documentation</li>
                        </ul>
                      </div>
    </th>
</tr>

<tr>
  <th></th>
  <th></th>
  <th></th>
  <th></th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/cells_by_motifs.txt" download>cells_by_motifs.txt</a></th>
  <th>01/11/2018</th>
  <th>3.4GB</th>
  <th>Aggregated motif scores for cells.
    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#cell-motifs-format" role="button" aria-expanded="false" aria-controls="cell-motifs-format">format details</a>
                      <div class="collapse" id="cell-motifs-format">
                        <ul>
                          <li><code>filter</code>: name of filter PWM used to scan accessible sites using TomTom</li>
                          <li><code>cell</code>: cell barcode (combined and corrected)</li>
                          <li><code>motif_score</code>: the total number of sites that had this matched this PWM</li>
                          <li><code>total_motifs</code>: total number of motifs detected in all the accessible sites for this cell</li>
                          <li><code>motif_activity</code>: shows the relative frequency of this motif relative with all the motifs scored for this cell (motif score/total motifs)</li>
                        </ul>
                      </div>
    </th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Basset/cells_by_motifs.rds" download>cells_by_motifs.rds</a></th>
  <th>01/11/2018</th>
  <th>674.4MB</th>
  <th>Same as above in RDS format.</th>
</tr>
                </tbody>
              </table>



<div class="hugespacer"></div>

          <h2>Annotation Enrichments</h2>

          <p>To aid in interpretation we have found it helpful to calculate gene set enrichments using peaks that are DA and have positive betas (are open). We report these enrichments for a number of different gene sets.</p>

          <div class= "panel panel-info">
            <div class= "panel-heading">
            <h3 class= "panel-title" data-toc-skip>Specificity Score Format</h3>
            </div>
            <div class= "panel-body">
            <p>The following columns are provided to report annotation enrichments for each cluster:</p>
            <table class = "table">
              <thead>
                <tr>
                  <th>Column</th>
                  <th>Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <th><code>cluster</code></th>
                  <th>Cluster assignment in initial t-SNE.</th>
                </tr>

                <tr>
                  <th><code>subset_cluster</code></th>
                  <th>Cluster assignment in the iterative t-SNE space.</th>
                </tr>

                <tr>
                  <th><code>term</code></th>
                  <th>Term for the gene set reported in the original gene set.</th>
                </tr>

                <tr>
                  <th><code>name</code></th>
                  <th>A cleaned version of the <code>term</code> column used in figures.</th>
                </tr>

                <tr>
                  <th><code>p.value</code></th>
                  <th>P-value of the reported enrichment.</th>
                </tr>

                <tr>
                  <th><code>Adjusted.p.value</code></th>
                  <th>P-value of the reported enrichment adjusted for multiple testing.</th>
                </tr>

                <tr>
                  <th><code>fold_change</code></th>
                  <th>Fold change of the reported enrichment.</th>
                </tr>

                <tr>
                  <th><code>gene_coverage</code></th>
                  <th>Fraction of the gene set covered in enrichment test.</th>
                </tr>

              </tbody>
            </table>
            </div>
          </div>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_all_pathways.txt" download>enrichment_all_pathways.txt</a></th>
  <th>01/16/2018</th>
  <th>25.8MB</th>
  <th>Enrichments for terms in <code>all pathways</code> gene sets in TSV format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_all_pathways.rds" download>enrichment_all_pathways.rds</a></th>
  <th>01/16/2018</th>
  <th>5.7MB</th>
  <th>Same as above in RDS format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_GO_bp.txt" download>enrichment_GO_bp.txt</a></th>
  <th>01/16/2018</th>
  <th>65.1MB</th>
  <th>Enrichments for terms in <code>GO biological processes</code> gene sets in TSV format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_GO_bp.rds" download>enrichment_GO_bp.rds</a></th>
  <th>01/16/2018</th>
  <th>12.3MB</th>
  <th>Same as above in RDS format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_mouse_phenotype.txt" download>enrichment_mouse_phenotype.txt</a></th>
  <th>01/16/2018</th>
  <th>16.3MB</th>
  <th>Enrichments for terms in <code>mouse phenotypes</code> gene sets in TSV format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_mouse_phenotype.rds" download>enrichment_mouse_phenotype.rds</a></th>
  <th>01/16/2018</th>
  <th>1.8MB</th>
  <th>Same as above in RDS format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_reactome.txt" download>enrichment_reactome.txt</a></th>
  <th>01/16/2018</th>
  <th>14.0MB</th>
  <th>Enrichments for terms in <code>reactome</code> gene sets in TSV format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/Annotation_Enrichments/enrichment_reactome.rds" download>enrichment_reactome.rds</a></th>
  <th>01/16/2018</th>
  <th>3.2MB</th>
  <th>Same as above in RDS format.</th>
</tr>
                </tbody>
              </table>



<div class="hugespacer"></div>

          <h2>GWAS <i>h</i><sup>2</sup> Enrichments</h2>
          <p>As described in the manuscript, we report enrichments in heritability (<i>h</i><sup>2</sup>) in DA peaks with positive betas for each cluster across many human traits as measured by GWAS.</p>

          <p>These enrichments are calculated using a tool called partitioned LD score regression (LDSC; <a href='https://www.nature.com/articles/ng.3404'>Finucane <i>et al.</i></a>; <a href='https://github.com/bulik/ldsc'>Github</a>). We also report the trained LDSC models and baseline model which could be used to calculate enrichments for any other trait given the appropriate summary statistics.</p>


            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/gwas_metadata.xlsx" download>gwas_metadata.xlsx</a></th>
                  <th>08/02/2018</th>
                  <th>27.2KB</th>
                  <th>A table with information about all GWAS (other than UKBB) used in our analysis and where summary statistics can be accessed.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#gwas-metadata-format" role="button" aria-expanded="false" aria-controls="gwas-metadata-format">format details</a>
                    <div class="collapse" id="gwas-metadata-format">
                      <ul>
                          <li><code>gwas_name</code>: the trait examined in the GWAS</li>
                          <li><code>data_url</code>: URL to the file summary statistics were obtained from</li>
                          <li><code>lead_author</code>: the lead author(s) of the publication</li>
                          <li><code>year</code>: year of publication</li>
                          <li><code>journal</code>: the journal the study was published in</li>
                          <li><code>study_url</code>: A URL to the publication associated with the dataset</li>
                          <li><code>other_notes</code>: any other comments about data from this study</li>
                      </ul>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/gwas.all_results.txt" download>gwas.all_results.txt</a></th>
                  <th>08/02/2018</th>
                  <th>538.6KB</th>
                  <th>Heritability enrichments for GWAS examined in manuscript (not UKBB) for each cluster in TSV format.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#gwas-format" role="button" aria-expanded="false" aria-controls="gwas-format">format details</a>
                    <div class="collapse" id="gwas-format">
                      <ul>
                          <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                          <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                          <li><code>gwas</code>: the phenotype studied in GWAS</li>
                          <li><code>h2</code>: the heritability estimated for the trait in question (<code>gwas</code> column)</li>
                          <li><code>h2_se</code>: standard error of <code>h2</code></li>
                          <li><code>snp_count</code>: The total SNPs reported to be used by LDSC in log files.</li>
                          <li><code>prop_snps</code>: The proportion of <code>snp_count</code> that fall into DA peaks for the specified cluster</li>
                          <li><code>prop_h2</code>: The proportion of total <code>h2</code> that the DA peaks account for.</li>
                          <li><code>coefficient</code>: The coefficient reported by the LDSC model for the DA peak membership term.</li>
                          <li><code>scaled_coefficient</code>: <code>coefficient</code> column divided by the per SNP heritability. Roughly interpretable as an enrichment</li>
                          <li><code>enrichment</code>: an enrichment calculated independent of the baseline enrichment</li>
                          <li><code>coefficient_zscore</code>: the z-score reported for the unscaled <code>coefficient</code></li>
                          <li><code>coefficient_std_error</code>:  standard error for <code>coefficient</code></li>
                          <li><code>coefficient_pval</code>: P-value for the coefficient being non-zero (enriched over baseline)</li>
                          <li><code>coefficient_qval</code>: <code>pvalue</code> adjusted for multiple testing</li>
                      </ul>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/gwas.clustered_matrix.txt" download>gwas.clustered_matrix.txt</a></th>
                  <th>08/02/2018</th>
                  <th>8.0KB</th>
                  <th>A matrix of -log10(qval) for any significant enrichments from above file in TSV format.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#gwas-matrix-format" role="button" aria-expanded="false" aria-controls="gwas-matrix-format">format details</a>
                    <div class="collapse" id="gwas-matrix-format">
                      <p>Each row is a trait (as indicated by <code>trait</code> column) and each additional column is a cluster of cells. Entries appear in order according to the hierarchical clustering that appears in the manuscript. Zero entries indicate enrichments that are not significant.</p>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th></th>
                  <th></th>
                  <th></th>
                  <th></th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/gwas.tissues.all_results.txt" download>gwas.tissues.all_results.txt</a></th>
                  <th>08/06/2018</th>
                  <th>100.1KB</th>
                  <th>Heritability enrichments for GWAS examined in manuscript (not UKBB) for peaks called from each tissue in TSV format.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#gwas-tissues-format" role="button" aria-expanded="false" aria-controls="gwas-tissues-format">format details</a>
                    <div class="collapse" id="gwas-tissues-format">
                      <ul>
                          <li><code>tissue</code>: the tissue (replicates are not collapsed)</li>
                          <li><code>gwas</code>: the phenotype studied in GWAS</li>
                          <li><code>h2</code>: the heritability estimated for the trait in question (<code>gwas</code> column)</li>
                          <li><code>h2_se</code>: standard error of <code>h2</code></li>
                          <li><code>snp_count</code>: The total SNPs reported to be used by LDSC in log files.</li>
                          <li><code>prop_snps</code>: The proportion of <code>snp_count</code> that fall into DA peaks for the specified cluster</li>
                          <li><code>prop_h2</code>: The proportion of total <code>h2</code> that the DA peaks account for.</li>
                          <li><code>coefficient</code>: The coefficient reported by the LDSC model for the DA peak membership term.</li>
                          <li><code>scaled_coefficient</code>: <code>coefficient</code> column divided by the per SNP heritability. Roughly interpretable as an enrichment</li>
                          <li><code>enrichment</code>: an enrichment calculated independent of the baseline enrichment</li>
                          <li><code>coefficient_zscore</code>: the z-score reported for the unscaled <code>coefficient</code></li>
                          <li><code>coefficient_std_error</code>:  standard error for <code>coefficient</code></li>
                          <li><code>coefficient_pval</code>: P-value for the coefficient being non-zero (enriched over baseline)</li>
                          <li><code>coefficient_qval</code>: <code>pvalue</code> adjusted for multiple testing</li>
                      </ul>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/gwas.tissues.clustered_matrix.txt" download>gwas.tissues.clustered_matrix.txt</a></th>
                  <th>08/06/2018</th>
                  <th>9.6KB</th>
                  <th>A matrix of -log10(qval) for any significant enrichments from above file in TSV format.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#gwas-tissues-matrix-format" role="button" aria-expanded="false" aria-controls="gwas-tissues-matrix-format">format details</a>
                    <div class="collapse" id="gwas-tissues-matrix-format">
                      <p>Each row is a trait (as indicated by <code>trait</code> column) and each additional column a tissue. Entries appear in order according to the hierarchical clustering that appears in the manuscript. Zero entries indicate enrichments that are not significant.</p>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th></th>
                  <th></th>
                  <th></th>
                  <th></th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/ukbb.all_results.txt" download>ukbb.all_results.txt</a></th>
                  <th>08/02/2018</th>
                  <th>9.4MB</th>
                  <th>Heritability enrichments for UKBB GWAS examined in manuscript for each cluster in TSV format.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#ukbb-format" role="button" aria-expanded="false" aria-controls="ukbb-format">format details</a>
                    <div class="collapse" id="ukbb-format">
                      <ul>
                          <li><code>cluster</code>: cluster assignment in initial t-SNE</li>
                          <li><code>subset_cluster</code>: cluster assignment in iterative t-SNE space</li>
                          <li><code>field</code>: the phenotype description provided by the Neale Lab</li>
                          <li><code>field_cleaned</code>: a (sometimes) shortened version of <code>field</code> to remove redundant text</li>
                          <li><code>field_code</code>: the non-descriptive ID for each <code>field</code> value</li>
                          <li><code>effective_n</code>: the effective sample size for this trait</li>
                          <li><code>h2</code>: the heritability estimated for the trait in question (<code>gwas</code> column)</li>
                          <li><code>h2_se</code>: standard error of <code>h2</code></li>
                          <li><code>snp_count</code>: The total SNPs reported to be used by LDSC in log files.</li>
                          <li><code>prop_snps</code>: The proportion of <code>snp_count</code> that fall into DA peaks for the specified cluster</li>
                          <li><code>prop_h2</code>: The proportion of total <code>h2</code> that the DA peaks account for.</li>
                          <li><code>coefficient</code>: The coefficient reported by the LDSC model for the DA peak membership term.</li>
                          <li><code>scaled_coefficient</code>: <code>coefficient</code> column divided by the per SNP heritability. Roughly interpretable as an enrichment</li>
                          <li><code>enrichment</code>: an enrichment calculated independent of the baseline enrichment</li>
                          <li><code>coefficient_zscore</code>: the z-score reported for the unscaled <code>coefficient</code></li>
                          <li><code>coefficient_std_error</code>:  standard error for <code>coefficient</code></li>
                          <li><code>coefficient_pval</code>: P-value for the coefficient being non-zero (enriched over baseline)</li>
                          <li><code>coefficient_qval</code>: <code>pvalue</code> adjusted for multiple testing</li>
                      </ul>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/ukbb.clustered_matrix.txt" download>ukbb.clustered_matrix.txt</a></th>
                  <th>08/02/2018</th>
                  <th>81.9KB</th>
                  <th>A matrix of -log10(qval) for any significant enrichments from above file in TSV format.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#ukbb-matrix-format" role="button" aria-expanded="false" aria-controls="ukbb-matrix-format">format details</a>
                    <div class="collapse" id="ukbb-matrix-format">
                      <p>Each row is a trait (as indicated by <code>trait</code> column) and each additional column a cluster in TSV format. Entries appear in order according to the hierarchical clustering that appears in the manuscript. Zero entries indicate enrichments that are not significant.</p>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/ukbb.subset.clustered_matrix.txt" download>ukbb.subset.clustered_matrix.txt</a></th>
                  <th>08/02/2018</th>
                  <th>45.0KB</th>
                  <th>A matrix of -log10(qval) for a subset of phenotypes (as shown in manuscript) in TSV format.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#ukbb-matrix-subset-format" role="button" aria-expanded="false" aria-controls="ukbb-matrix-subset-format">format details</a>
                    <div class="collapse" id="ukbb-matrix-subset-format">
                      <p>Each row is a trait (as indicated by <code>trait</code> column) and each additional column a cluster. Entries appear in order according to the hierarchical clustering that appears in the manuscript. Zero entries indicate enrichments that are not significant.</p>
                    </div>
                  </th>
                </tr>

                <tr>
                  <th></th>
                  <th></th>
                  <th></th>
                  <th></th>
                </tr>

                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/gwas_h2_enrichments/ld_score_regression_models.tar.gz" download>ld_score_regression_models.tar.gz</a></th>
                  <th>08/02/2018</th>
                  <th>9.7GB</th>
                  <th>Set of models for each cluster and a baseline model for comparison trained with LDSC.
                    <a class="btn btn-info btn-xs" data-toggle="collapse" href="#ldsc_model-format" role="button" aria-expanded="false" aria-controls="ldsc_model-format">format details</a>
                    <div class="collapse" id="ldsc_model-format">
                      <p>When unpacked with <code>tar -xzvf ld_score_regression_models.tar.gz</code>, will contain <code>cluster_models</code> and <code>baseline_model</code> subdirectories. These contain the models trained using DA peaks from each cluster and the baseline model to compare against, respectively.</p>

                      <p>Note that <code>cluster_models</code> will contain one file per cluster per chromosome, so there will be a large number of files in this subdirectory.</p>

                      <p>These files may then be used in conjuction with the final step of LDSC (see <a href="https://github.com/bulik/ldsc">Github</a> for usage and format descriptions) to calculate enrichments of heritability within the DA peaks for a given cluster for summary statistics from any GWAS.</p>

                    </div>
                  </th>
                </tbody>
              </table>


          <h2>BAM Files</h2>

          <p>While we provide some raw data on <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111586">GEO (GSE111586)</a>, we also provide BAM files of the sequences aligned to <code>mm9</code> here in case users would like to use them for their own pipelines or methods development. Below we provide one file per tissue, named by their <code>tissue.replicate</code> ID as specified in the <code>tissue.replicate</code> column of the <code>cell_metadata.txt</code> file in the <a href='#metadata'>metadata section</a> above. This means there will be two files for tissues where we performed a replicate and a single file for all other tissues (in addition to BAM index files).</p>

          <div class= "panel panel-warning">
            <div class= "panel-heading">
            <h3 class= "panel-title" data-toc-skip>Important: BAM Details</h3>
            </div>
            <div class= "panel-body">
              <ul>
                <li>Each read is assigned to a cell ID (the sequence specified in the <code>cell</code> column of the same metadata file mentioned above). This is encoded in the read name as <code>cellid:otherinfo</code>, so the sequence before the colon is the corrected cell barcode sequence for the read.</li>

                <li>Reads are already deduplicated.</li>

                <li>There will be cell IDs that do not appear in our final set of cells, as data is a superset of what ultimately passes our QC steps.</li>

                <li>Files may not download correctly in Chrome (and other web browsers), but they can easily be downloaded with <code>wget</code> or <code>curl</code>, by right clicking and copying the link address. For example:</li>

                <pre><code class="language-bash" data-lang="bash">wget http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/BoneMarrow_62016.bam</code></pre>
              </ul>
            </div>
          </div>

<table class = "table">
  <thead>
    <tr>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th width = "50%">Description</th>
    </tr>
  </thead>
  <tbody>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/BoneMarrow_62016.bam" download>BoneMarrow_62016.bam</a></th>
  <th>07/04/2017</th>
  <th>7.3GB</th>
  <th>BAM file for BoneMarrow_62016.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/BoneMarrow_62016.bam.bai" download>BoneMarrow_62016.bam.bai</a></th>
  <th>07/12/2017</th>
  <th>2.9MB</th>
  <th>BAM index for BoneMarrow_62016.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/BoneMarrow_62216.bam" download>BoneMarrow_62216.bam</a></th>
  <th>07/04/2017</th>
  <th>6.6GB</th>
  <th>BAM file for BoneMarrow_62216.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/BoneMarrow_62216.bam.bai" download>BoneMarrow_62216.bam.bai</a></th>
  <th>07/12/2017</th>
  <th>2.5MB</th>
  <th>BAM index for BoneMarrow_62216.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Cerebellum_62216.bam" download>Cerebellum_62216.bam</a></th>
  <th>07/04/2017</th>
  <th>2.1GB</th>
  <th>BAM file for Cerebellum_62216.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Cerebellum_62216.bam.bai" download>Cerebellum_62216.bam.bai</a></th>
  <th>07/12/2017</th>
  <th>2.0MB</th>
  <th>BAM index for Cerebellum_62216.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/LargeIntestineA_62816.bam" download>LargeIntestineA_62816.bam</a></th>
  <th>07/04/2017</th>
  <th>4.8GB</th>
  <th>BAM file for LargeIntestineA_62816.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/LargeIntestineA_62816.bam.bai" download>LargeIntestineA_62816.bam.bai</a></th>
  <th>07/12/2017</th>
  <th>2.1MB</th>
  <th>BAM index for LargeIntestineA_62816.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/LargeIntestineB_62816.bam" download>LargeIntestineB_62816.bam</a></th>
  <th>07/04/2017</th>
  <th>8.9GB</th>
  <th>BAM file for LargeIntestineB_62816.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/LargeIntestineB_62816.bam.bai" download>LargeIntestineB_62816.bam.bai</a></th>
  <th>07/12/2017</th>
  <th>3.3MB</th>
  <th>BAM index for LargeIntestineB_62816.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/HeartA_62816.bam" download>HeartA_62816.bam</a></th>
  <th>07/04/2017</th>
  <th>9.9GB</th>
  <th>BAM file for HeartA_62816.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/HeartA_62816.bam.bai" download>HeartA_62816.bam.bai</a></th>
  <th>07/12/2017</th>
  <th>3.4MB</th>
  <th>BAM index for HeartA_62816.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/SmallIntestine_62816.bam" download>SmallIntestine_62816.bam</a></th>
  <th>07/04/2017</th>
  <th>9.3GB</th>
  <th>BAM file for SmallIntestine_62816.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/SmallIntestine_62816.bam.bai" download>SmallIntestine_62816.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>3.9MB</th>
  <th>BAM index for SmallIntestine_62816.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Kidney_62016.bam" download>Kidney_62016.bam</a></th>
  <th>07/04/2017</th>
  <th>8.2GB</th>
  <th>BAM file for Kidney_62016.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Kidney_62016.bam.bai" download>Kidney_62016.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>2.9MB</th>
  <th>BAM index for Kidney_62016.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Liver_62016.bam" download>Liver_62016.bam</a></th>
  <th>07/04/2017</th>
  <th>8.1GB</th>
  <th>BAM file for Liver_62016.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Liver_62016.bam.bai" download>Liver_62016.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>2.8MB</th>
  <th>BAM index for Liver_62016.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Lung1_62216.bam" download>Lung1_62216.bam</a></th>
  <th>07/04/2017</th>
  <th>5.4GB</th>
  <th>BAM file for Lung1_62216.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Lung1_62216.bam.bai" download>Lung1_62216.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>2.4MB</th>
  <th>BAM index for Lung1_62216.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Lung2_62216.bam" download>Lung2_62216.bam</a></th>
  <th>07/04/2017</th>
  <th>6.3GB</th>
  <th>BAM file for Lung2_62216.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Lung2_62216.bam.bai" download>Lung2_62216.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>2.4MB</th>
  <th>BAM index for Lung2_62216.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/PreFrontalCortex_62216.bam" download>PreFrontalCortex_62216.bam</a></th>
  <th>07/04/2017</th>
  <th>11.1GB</th>
  <th>BAM file for PreFrontalCortex_62216.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/PreFrontalCortex_62216.bam.bai" download>PreFrontalCortex_62216.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>4.0MB</th>
  <th>BAM index for PreFrontalCortex_62216.</th>
</tr>

 <tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Spleen_62016.bam" download>Spleen_62016.bam</a></th>
  <th>07/04/2017</th>
  <th>5.0GB</th>
  <th>BAM file for Spleen_62016.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Spleen_62016.bam.bai" download>Spleen_62016.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>2.5MB</th>
  <th>BAM index for Spleen_62016.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Testes_62016.bam" download>Testes_62016.bam</a></th>
  <th>07/04/2017</th>
  <th>12.5GB</th>
  <th>BAM file for Testes_62016.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Testes_62016.bam.bai" download>Testes_62016.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>5.7MB</th>
  <th>BAM index for Testes_62016.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Thymus_62016.bam" download>Thymus_62016.bam</a></th>
  <th>07/04/2017</th>
  <th>10.2GB</th>
  <th>BAM file for Thymus_62016.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/Thymus_62016.bam.bai" download>Thymus_62016.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>3.4MB</th>
  <th>BAM index for Thymus_62016.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/WholeBrainA_62216.bam" download>
WholeBrainA_62216.bam</a></th>
  <th>07/04/2017</th>
  <th>8.9GB</th>
  <th>BAM file for WholeBrainA_62216.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/WholeBrainA_62216.bam.bai" download>WholeBrainA_62216.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>3.4MB</th>
  <th>BAM index for WholeBrainA_62216.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/WholeBrainA_62816.bam" download>
WholeBrainA_62816.bam</a></th>
  <th>07/04/2017</th>
  <th>6.4GB</th>
  <th>BAM file for WholeBrainA_62816.</th>
</tr>
<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bams/WholeBrainA_62816.bam.bai" download>WholeBrainA_62816.bam.bai</a></th>
  <th>07/13/2017</th>
  <th>2.7MB</th>
  <th>BAM index for WholeBrainA_62816.</th>
</tr>


    </tbody>
  </table>


  <div class="hugespacer"></div>

  <h2>UCSC Trackhub and Bigwigs</h2>
  <p>We also provide bigWig files and a UCSC trackhub to visualize aggregated pseudo-bulk ATAC-seq profiles for the cells from each cluster. Note that for the smallest clusters, the data will appear fairly sparse even in aggregate at any single locus. In general, we prefer methods for assessing differential accessibility or specificity computationally over visual inspection, although viewing tracks is often useful to get a sense for the data at a given locus.</p>

  <h3>UCSC Trackhub</h3>
  <p>You may access our <a href='http://genome.ucsc.edu/cgi-bin/hgTracks?db=mm9&hubUrl=http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/trackhub/hub.txt'>UCSC trackhub here</a>. By default the hub will contain a track at the top called  <code>_All_Peak_Calls</code>, which annotates regions that we called within LSI clusters for each tissue (see Methods). These peaks were used as our features for all downstream analysis.</p>

  <p>The trackhub will also contain a track for each cluster in the dataset named according to the convention <code>cell_label-id</code>, where <code>cell_label</code> and <code>id</code> are defined in the same way as they are in our <code>cell_metadata.txt</code> file <a href='#metadata'>above</a>. Spaces and periods in cell labels have been removed or replaced as necessary.</p>

   <div class= "panel panel-warning">
    <div class= "panel-heading">
    <h3 class= "panel-title" data-toc-skip>Important</h3>
  </div>
    <div class= "panel-body">

    <p>We have generated these tracks using <a href='https://deeptools.readthedocs.io/en/develop/'>DeepTools3</a> and <code>CPM</code> or <code>Counts Per Million</code> normalization with using the <code>bamCoverage</code> command with arguments <code>-bs 1 --normalizeUsing CPM --skipNAs</code>. The default range displayed on UCSC is 0 to 4 for all tracks, which we find generally works well. However, it is possible that it may need to be adjusted in some cases.</p>
    </div>
  </div>

  <div class="spacer"></div>

  <h3>BigWig Files</h3>
  <p>In case you would like access to the files used to make the trackhub above, we provide them for download below. Each file is named in the same manner as described above with a <code>.bw</code> or <code>.bb</code> extension.</p>

<table class = "table">
  <thead>
    <tr>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th width="50%">Description</th>
    </tr>
  </thead>
  <tbody>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/master_peaks_track.bb" download>master_peaks_track.bb</a></th>
      <th>08/14/2018</th>
      <th>3.9MB</th>
      <th>bigBed formatted file with our master set of peak calls.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Activated_B_cells-clusters_4-cluster_4.bw" download>Activated_B_cells-clusters_4-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>77.4MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Alveolar_macrophages-clusters_17-cluster_2.bw" download>Alveolar_macrophages-clusters_17-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>166.7MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Astrocytes-clusters_19-cluster_1.bw" download>Astrocytes-clusters_19-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>144.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Astrocytes-clusters_19-cluster_2.bw" download>Astrocytes-clusters_19-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>69.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Astrocytes-clusters_19-cluster_3.bw" download>Astrocytes-clusters_19-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>32.4MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Astrocytes-clusters_19-cluster_4.bw" download>Astrocytes-clusters_19-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>29.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/B_cells-clusters_16-cluster_1.bw" download>B_cells-clusters_16-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>323.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/B_cells-clusters_4-cluster_1.bw" download>B_cells-clusters_4-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>168.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/B_cells-clusters_4-cluster_2.bw" download>B_cells-clusters_4-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>178.3MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/B_cells-clusters_4-cluster_3.bw" download>B_cells-clusters_4-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>350.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Cardiomyocytes-clusters_7-cluster_1.bw" download>Cardiomyocytes-clusters_7-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>960.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Cerebellar_granule_cells-clusters_8-cluster_1.bw" download>Cerebellar_granule_cells-clusters_8-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>331.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Cerebellar_granule_cells-clusters_8-cluster_2.bw" download>Cerebellar_granule_cells-clusters_8-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>245.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collecting_duct-clusters_18-cluster_5.bw" download>Collecting_duct-clusters_18-cluster_5.bw</a></th>
      <th>08/14/2018</th>
      <th>42.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collisions-clusters_17-cluster_4.bw" download>Collisions-clusters_17-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>45.8MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collisions-clusters_17-cluster_5.bw" download>Collisions-clusters_17-cluster_5.bw</a></th>
      <th>08/14/2018</th>
      <th>82.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collisions-clusters_26-cluster_2.bw" download>Collisions-clusters_26-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>56.3MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collisions-clusters_26-cluster_3.bw" download>Collisions-clusters_26-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>100.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collisions-clusters_27-cluster_2.bw" download>Collisions-clusters_27-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>176.9MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collisions-clusters_27-cluster_3.bw" download>Collisions-clusters_27-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>56.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Collisions-clusters_30-cluster_3.bw" download>Collisions-clusters_30-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>19.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/DCT_CD-clusters_18-cluster_1.bw" download>DCT_CD-clusters_18-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>138.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Dendritic_cells-clusters_17-cluster_1.bw" download>Dendritic_cells-clusters_17-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>101.6MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Dendritic_cells-clusters_17-cluster_3.bw" download>Dendritic_cells-clusters_17-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>123.7MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Distal_convoluted_tubule-clusters_18-cluster_4.bw" download>Distal_convoluted_tubule-clusters_18-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>90.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_II_cells-clusters_23-cluster_1.bw" download>Endothelial_II_cells-clusters_23-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>282.6MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_II_cells-clusters_25-cluster_2.bw" download>Endothelial_II_cells-clusters_25-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>50.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_II_cells-clusters_25-cluster_3.bw" download>Endothelial_II_cells-clusters_25-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>14.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_II_cells-clusters_9-cluster_2.bw" download>Endothelial_II_cells-clusters_9-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>168.3MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_II_cells-clusters_9-cluster_3.bw" download>Endothelial_II_cells-clusters_9-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>45.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_I_(glomerular)-clusters_22-cluster_2.bw" download>Endothelial_I_(glomerular)-clusters_22-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>67.7MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_I_cells-clusters_22-cluster_1.bw" download>Endothelial_I_cells-clusters_22-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>83.4MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_I_cells-clusters_22-cluster_3.bw" download>Endothelial_I_cells-clusters_22-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>21.8MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Endothelial_I_cells-clusters_22-cluster_4.bw" download>Endothelial_I_cells-clusters_22-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>20.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Enterocytes-clusters_6-cluster_1.bw" download>Enterocytes-clusters_6-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>1.2GB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Erythroblasts-clusters_13-cluster_1.bw" download>Erythroblasts-clusters_13-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>483.1MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Ex_neurons_CPN-clusters_5-cluster_1.bw" download>Ex_neurons_CPN-clusters_5-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>736.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Ex_neurons_CThPN-clusters_5-cluster_3.bw" download>Ex_neurons_CThPN-clusters_5-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>405.7MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Ex_neurons_CThPN-clusters_5-cluster_4.bw" download>Ex_neurons_CThPN-clusters_5-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>441.8MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Ex_neurons_SCPN-clusters_29-cluster_1.bw" download>Ex_neurons_SCPN-clusters_29-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>108.3MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Ex_neurons_SCPN-clusters_5-cluster_2.bw" download>Ex_neurons_SCPN-clusters_5-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>609.6MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Hematopoietic_progenitors-clusters_10-cluster_1.bw" download>Hematopoietic_progenitors-clusters_10-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>1.1GB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Hepatocytes-clusters_3-cluster_1.bw" download>Hepatocytes-clusters_3-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>1.1GB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Immature_B_cells-clusters_28-cluster_1.bw" download>Immature_B_cells-clusters_28-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>41.3MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Immature_B_cells-clusters_28-cluster_2.bw" download>Immature_B_cells-clusters_28-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>58.7MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Inhibitory_neurons-clusters_15-cluster_1.bw" download>Inhibitory_neurons-clusters_15-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>585.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Inhibitory_neurons-clusters_15-cluster_2.bw" download>Inhibitory_neurons-clusters_15-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>227.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Inhibitory_neurons-clusters_5-cluster_5.bw" download>Inhibitory_neurons-clusters_5-cluster_5.bw</a></th>
      <th>08/14/2018</th>
      <th>156.9MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Loop_of_henle-clusters_18-cluster_2.bw" download>Loop_of_henle-clusters_18-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>126.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Loop_of_henle-clusters_18-cluster_3.bw" download>Loop_of_henle-clusters_18-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>69.8MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Macrophages-clusters_16-cluster_2.bw" download>Macrophages-clusters_16-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>181.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Microglia-clusters_16-cluster_3.bw" download>Microglia-clusters_16-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>81.1MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Monocytes-clusters_24-cluster_1.bw" download>Monocytes-clusters_24-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>132.3MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Monocytes-clusters_24-cluster_2.bw" download>Monocytes-clusters_24-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>151.4MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/NK_cells-clusters_12-cluster_3.bw" download>NK_cells-clusters_12-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>63.8MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Oligodendrocytes-clusters_21-cluster_1.bw" download>Oligodendrocytes-clusters_21-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>124.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Oligodendrocytes-clusters_21-cluster_2.bw" download>Oligodendrocytes-clusters_21-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>84.1MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Podocytes-clusters_25-cluster_1.bw" download>Podocytes-clusters_25-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>89.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Proximal_tubule-clusters_11-cluster_1.bw" download>Proximal_tubule-clusters_11-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>137.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Proximal_tubule-clusters_11-cluster_2.bw" download>Proximal_tubule-clusters_11-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>222.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Proximal_tubule-clusters_11-cluster_3.bw" download>Proximal_tubule-clusters_11-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>168.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Proximal_tubule-clusters_11-cluster_5.bw" download>Proximal_tubule-clusters_11-cluster_5.bw</a></th>
      <th>08/14/2018</th>
      <th>192.6MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Proximal_tubule_S3-clusters_11-cluster_4.bw" download>Proximal_tubule_S3-clusters_11-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>252.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Purkinje_cells-clusters_27-cluster_1.bw" download>Purkinje_cells-clusters_27-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>95.1MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Regulatory_T_cells-clusters_12-cluster_2.bw" download>Regulatory_T_cells-clusters_12-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>101.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/SOM+_Interneurons-clusters_15-cluster_3.bw" download>SOM+_Interneurons-clusters_15-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>222.1MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Sperm-clusters_14-cluster_1.bw" download>Sperm-clusters_14-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>274.3MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Sperm-clusters_14-cluster_2.bw" download>Sperm-clusters_14-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>137.4MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Sperm-clusters_14-cluster_3.bw" download>Sperm-clusters_14-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>140.9MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/T_cells-clusters_12-cluster_1.bw" download>T_cells-clusters_12-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>285.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/T_cells-clusters_12-cluster_4.bw" download>T_cells-clusters_12-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>90.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/T_cells-clusters_12-cluster_5.bw" download>T_cells-clusters_12-cluster_5.bw</a></th>
      <th>08/14/2018</th>
      <th>27.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/T_cells-clusters_2-cluster_1.bw" download>T_cells-clusters_2-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>1.1GB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/T_cells-clusters_26-cluster_1.bw" download>T_cells-clusters_26-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>78.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Type_II_pneumocytes-clusters_30-cluster_1.bw" download>Type_II_pneumocytes-clusters_30-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>37.0MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Type_I_pneumocytes-clusters_20-cluster_1.bw" download>Type_I_pneumocytes-clusters_20-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>282.6MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_1-cluster_1.bw" download>Unknown-clusters_1-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>1.3GB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_1-cluster_2.bw" download>Unknown-clusters_1-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>598.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_1-cluster_3.bw" download>Unknown-clusters_1-cluster_3.bw</a></th>
      <th>08/14/2018</th>
      <th>369.5MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_23-cluster_2.bw" download>Unknown-clusters_23-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>40.8MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_30-cluster_2.bw" download>Unknown-clusters_30-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>66.9MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_30-cluster_4.bw" download>Unknown-clusters_30-cluster_4.bw</a></th>
      <th>08/14/2018</th>
      <th>16.9MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_5-cluster_6.bw" download>Unknown-clusters_5-cluster_6.bw</a></th>
      <th>08/14/2018</th>
      <th>102.6MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_7-cluster_2.bw" download>Unknown-clusters_7-cluster_2.bw</a></th>
      <th>08/14/2018</th>
      <th>45.2MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>

    <tr>
      <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/bigwigs/Unknown-clusters_9-cluster_1.bw" download>Unknown-clusters_9-cluster_1.bw</a></th>
      <th>08/14/2018</th>
      <th>613.1MB</th>
      <th>bigWig formated track for cluster.</th>
    </tr>


  </tbody>
</table>

</div>

</div>
</div>

<br>
<br>
<br>
<br>

{% include footer.html %}

</body>

</html>

