---
layout: vanilla
permalink: /data/
description: "Downloads for Mouse Atlas."
modified: 2018-01-13
tags: [worm, manual, vignette, atlas]
---
<!DOCTYPE html>
<html lang= "{{ page.lang | default: site.lang | default: "en" }}">

  {% include head.html %}

  <!-- See here for how to use the TOC control:
  https://afeld.github.io/bootstrap-toc/ -->
  <body  data-spy= "scroll" data-target= "#toc">
    {% include header.html %}

    <div class= "container">
      <div class= "row">
        <!-- sidebar, which will move to the top on a small screen -->
        <div class= "col-sm-4 sidenav">
          <nav id= "toc" data-spy= "affix" data-toggle= "toc"></nav>
        </div>
        <!-- main content area -->
        <div class= "col-sm-8">
  <div class="hugespacer"></div>
          <h2>ATAC Matrices</h2>

          <p>Similar to sc-RNA-seq, sci-ATAC-seq data is typically analyzed in sparse <code>peak</code> (row) by <code>cell (column)</code> matrices. The first set we provide are binarized counts. The second set has rare peaks filtered out and is then normalized with TFIDF to allow for input to  PCA/TSNE, for example. Note that only cells in our final QC filtered set are included.</p>

          <p>See <a href="{{ site.baseurl }}/docs/">tutorials</a> for examples of how to read these formats into <code>R</code> or <code>python</code> along with documentation on lots of other downstream analysis.</p>

          <div class= "panel panel-info">
            <div class= "panel-heading">
              <h3 class= "panel-title" data-toc-skip>Matrix Formats</h3>
            </div>
            <div class= "panel-body">
            <p>In general, we provide two formats for all matrices:</p>
            <table class = "table">
              <thead>
                <tr>
                  <th>File Type</th>
                  <th>Description</th>
                </tr>
              </thead>

              <tbody>
                <tr>
                  <th><code>.mtx.gz</code></th>
                  <th><p>Same as format generated by 10X Genomics <code>cellranger</code> pipeline (matrix market format). <code>.mtx</code> files are provided with <code>.txt</code> files containing peak and cell IDs that correspond to the rows and columns of the matrix, respectively.</p></th>
                </tr>

                <tr>
                  <th><code>.rds</code></th>
                  <th><p>RDS format that can be read into R directly with <code>readRDS</code> function.</p></th>
                </tr>
              </tbody>
            </table>
            </div>
          </div>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filte
red.mtx.gz" download>atac_matrix.binary.qc_filtered.mtx.gz</a></th>
  <th>08/02/18</th>
  <th>1.1GB</th>
  <th>Binarized peak by cell matrix in matrix market format.</th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filtered.cells.txt" download>atac_matrix.binary.qc_filtered.cells.txt</a></th>
  <th>08/02/18</th>
  <th>2.9MB</th>
  <th>Cell IDs (columns) of binarized matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filtered.peaks.txt" download>atac_matrix.binary.qc_filtered.peaks.txt</a></th>
  <th>08/02/18</th>
  <th>10.0MB</th>
  <th>Peak IDs (rows) of binarized matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.binary.qc_filtered.rds" download>atac_matrix.binary.qc_filtered.rds</a></th>
  <th>08/02/18</th>
  <th>1.0GB</th>
  <th>Binarized peak by cell matrix in RDS format.</th>
</tr>

<tr>
  <th></th>
  <th></th>
  <th></th>
  <th></th>
</tr>


<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filter
ed.mtx.gz" download>atac_matrix.tfidf.qc_filtered.mtx.gz</a></th>
  <th>08/02/18</th>
  <th>3.4GB</th>
  <th>TFIDF normalized peak by cell matrix in matrix market format.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filtered.cells.txt" download>atac_matrix.tfidf.qc_filtered.cells.txt</a></th>
  <th>08/02/18</th>
  <th>2.9MB</th>
  <th>Cell IDs (columns) of TFIDF matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filtered.peaks.txt" download>atac_matrix.tfidf.qc_filtered.peaks.txt</a></th>
  <th>08/02/18</th>
  <th>3.8MB</th>
  <th>Peak IDs (rows) of TFIDF matrix.</th>
</tr>

<tr>
  <th><a href = "http://krishna.gs.washington.edu/content/members/ajh24/mouse_atlas_data_release/matrices/atac_matrix.tfidf.qc_filtered.rds" download>atac_matrix.tfidf.qc_filtered.rds</a></th>
  <th>08/02/18</th>
  <th>5.4GB</th>
  <th>TFIDF normalized peak by cell matrix in RDS format.</th>
</tr>

                </tbody>
              </table>




<div class="hugespacer"></div>

          <h2>Activity Score Matrices</h2>
          <p>We also report "gene activity scores", where a single number is calculated based on a weighted combination of proximal and distal sites for each gene (see manuscript for details; both quantitative and binarized calculations provided below). Unlike the ATAC matrices above, these are in <code>gene (row)</code> by <code>cell (column)</code> format.</p>

         <div class= "panel panel-warning">
            <div class= "panel-heading">
            <h3 class= "panel-title" data-toc-skip>Important: Size Factor Normalization</h3>
            </div>
            <div class= "panel-body">
<p>Quantitative scores provided here are not normalized by size factors, so you may apply size factor normalization to these values if needed.</p>
            </div>
          </div>

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>



  </tbody>
</table>

</div>

</div>
</div>

<br>
<br>
<br>
<br>

{% include footer.html %}

</body>

</html>

